src/components/Error.tsx

```diff
@@ -0,0 +1,38 @@
+import { useContext } from 'react';
+import cn from 'classnames';
+import { TodosContext } from '../context/TodoContext';
+
+// enum ErrorMessages {
+//   'Unable to load todos',
+//   'Title should not be empty',
+//   'Unable to add a todo',
+//   'Unable to delete a todo',
+//   'Unable to update a todo',
+// }
```

Comment:
every error message should have name 
```suggestion
// enum ErrorMessages {
//   'Unable to load todos',
//   'Title should not be empty',
//   'Unable to add a todo',
//   'Unable to delete a todo',
//   'Unable to update a todo',
// }
export enum ErrorType {
  None = '',
  Add = 'Unable to add a todo',
  Delete = 'Unable to delete a todo',
  Update = 'Unable to update a todo',
  Load = 'Unable to load todo',
  Empty = 'Title can not be empty',
}
```